SQLProject.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+function getSorter(){
```

Comment:
I think you can get rid of this file since it seems to existing in the include folder now